addons/sitting/functions/fnc_getRandomAnimation.sqf

```diff
@@ -15,30 +15,43 @@
  * Public: No
  */
 
+params [["_object", objNull, [objNull]]];
+// Original function had no parameters, so add default for BWC
+
+private _animations = [];
+if !(isNull configOf _object) then {
+    // configOf objNull returns CONFIG-NULL
+    _animations = getArray (configOf _object >> QGVAR(animations));
```

Comment:
Comments are obvious imo, no need for them.
```suggestion

private _animations = [];

if (!isNull _object) then {
    _animations = getArray (configOf _object >> QGVAR(animations));
```

Reply:
Only reason I did configOf was because I thought it might cause weird behavior if objNull was used (i.e. no param passed)